web/src/pages/NotificationsPage.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <h3 class="text-h3 mb-4">Notifications</h3>
+
+  <AppCard>
+    <v-btn-toggle
+      v-model="showFilter"
+      color="info"
+      group
+      mandatory
+      style="border: 1px #b0b0b0 solid !important; border-radius: 6px"
+      @update:model-value="page = 1"
+    >
+      <v-btn
+        value="All"
+        style="width: 100px; border-left: 1px #b0b0b0 solid"
+        text="All"
+      />
+      <v-btn
+        value="Unread"
+        style="width: 100px; border-left: 1px #b0b0b0 solid"
+        text="Unread"
+      />
+    </v-btn-toggle>
+
+    <v-list
+      class="py-0 mt-4"
+      lines="two"
+    >
+      <v-data-iterator :items="notifications">
```

Comment:
Data iterators are just a data table server but with a list instead of a table.
With that said, I'd make a NotificationsDataIterator component and make use of the data iterator build in pagination. I also think it's necessary to use a "route query sufix" on the component. See https://github.com/icefoganalytics/traditional-knowledge/blob/main/web/src/components/groups/GroupsEditDataTableServer.vue